display.go

```diff
@@ -117,7 +117,7 @@ func (ts *TaskSet) renderTable(truncate bool) error {
 		table.Render()
 
 		if truncate && maxTasks < total {
-			fmt.Printf("\n%v tasks, truncated to %v lines.\n", total, h)
+			fmt.Printf("\n%v tasks, truncated to %v lines.\n", total, maxTasks)
```

Comment:
the message should probably also be changed to something that says tasks instead of lines, maybe `fmt.Printf("\n%v tasks, showing to %v.\n", total, maxTasks)`

Reply:
What do you think about  `fmt.Printf("\n%v tasks, %v of them displayed.\n", total, maxTasks)` ?

Reply:
`of them` is probably superfluous, otherwise sounds good!

Reply:
Yeah. You're right. It is superfluous. :)
The last change in the pull request is with 
`fmt.Printf("\n%v/%v tasks shown.\n", maxTasks, total)`

Reply:
Thanks, will merge after work.